### ISSUE TITLE
Volume is freaking out

### ISSUE THREAD

**Issue by mathieuhays** [jaedb/Iris#291]
## Expected behavior
When the volume is set, the volume stays at the volume wanted.

## Actual behavior
The volume keeps changing randomly, going up and down for a few seconds.

## Steps to replicate
I haven't manage to replicate this consistently. It usually happens after the player stayed idle for a bit.
Changing the volume can trigger this behaviour. Starting a playlist does it too.
I have experienced this for a long time (even on previous versions).

#### Operating systems affected
Player viewed on MacOs 10.13.4
Mopidy installed on Raspbian (one of the latest version as of 01/06/18)

#### Browsers affected
I personally experience this on Firefox but I believe my collegues also experience that on Chrome.

#### Screen sizes affected
- [*] Huge (ie desktop)
- [*] Large (ie laptop)
- [ ] Medium (ie slim laptop/tablet)
- [ ] Small (phone)

## Additional details, debug data, logs, etc
mopidy log:
```
2018-06-01 09:00:41,263 INFO [522:HttpServer] tornado.access: 200 GET /iris/ (127.0.0.1) 10.47ms
2018-06-01 09:00:41,398 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.js?v=1527585609 (127.0.0.1) 38.50ms
2018-06-01 09:00:41,432 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.css?v=1527585609 (127.0.0.1) 26.27ms
2018-06-01 09:00:41,808 INFO [522:HttpServer] tornado.access: 101 GET /mopidy/ws/ (192.168.0.18) 3.94ms
2018-06-01 09:00:41,818 INFO [522:HttpServer] tornado.access: 101 GET /iris/ws/ (192.168.0.18) 3.33ms
2018-06-01 09:02:50,646 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.css?v=1527585609 (127.0.0.1) 32.23ms
2018-06-01 09:02:50,929 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.css.map (127.0.0.1) 5.96ms
2018-06-01 09:02:58,900 INFO [522:HttpServer] tornado.access: 200 GET /iris/ (127.0.0.1) 7.33ms
2018-06-01 09:02:59,108 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.js?v=1527585609 (127.0.0.1) 35.26ms
2018-06-01 09:02:59,140 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.css?v=1527585609 (127.0.0.1) 23.78ms
2018-06-01 09:02:59,457 INFO [522:HttpServer] tornado.access: 101 GET /mopidy/ws/ (192.168.0.18) 3.86ms
2018-06-01 09:02:59,467 INFO [522:HttpServer] tornado.access: 101 GET /iris/ws/ (192.168.0.18) 3.25ms
2018-06-01 09:03:31,300 INFO [522:HttpServer] tornado.access: 200 GET /iris/assets/backgrounds/settings.jpg (127.0.0.1) 10.08ms
2018-06-01 09:03:49,778 INFO [522:HttpServer] tornado.access: 200 GET /iris/app.min.js.map (127.0.0.1) 236.04ms
2018-06-01 09:16:25,972 INFO [522:HttpServer] tornado.access: 200 GET /iris/assets/backgrounds/settings.jpg (127.0.0.1) 9.52ms
```
Nothing appeared in the console of the browser. I just turned on action logging in the debugging section so once I manage to experience that issue again I'll send that through.

## Before you submit
- [ ] Yes, I have tested this issue thoroughly
- [ ] Yes, I have attached screenshots and/or screencast of the issue in action
- [ ] Yes, I have included any relevant debug data and error logs

**Comment by jaedb** [comment]
Sounds like a very unusual issue! It might sound daft, but perhaps you have a user on your network that is fiddling around? You could check to make sure you're the only client connected to Iris under _Settings > Advanced > Connections_.

One other thing you could try is installing `ncmpcpp`. It's a great command-line client for MPD. You could try opening this and seeing if the issue is caused when you have an Iris client connected/disconnected.

**Comment by mathieuhays** [comment]
> Sounds like a very unusual issue! It might sound daft, but perhaps you have a user on your network that is fiddling around? You could check to make sure you're the only client connected to Iris under Settings > Advanced > Connections.

Understandable but unfortunately that's not the case. The last time it happened I had just restarted the pi, I was the only one to know the new IP and the only one connected.

As I said it tends to happen after the player has been idle for a while.
It has been happenning for months, about once a day or so.
I've been the only one manipulating it for the past days and it didn't happen since my report.

What might be happening is user A changes the volume. Player stops itself. An hour later user B starts new playlist, volume has strange behaviour as the playlist starts.

When that behaviour starts, the volume seems to increase consistently. Since the music gets super loud we always tend to click on the lowests volumes on the control which might create that pattern where it goes up and down.

Something I did notice since I enabled action logging. When a `MOPIDY_SET_VOLUME` action is triggered it usually generates multiple `MOPIDY_VOLUME` back. Is that expected?

![screen shot 2018-06-05 at 11 06 55](https://user-images.githubusercontent.com/5703113/40969758-a904839e-68b0-11e8-9771-c78e03ec7491.png)

> One other thing you could try is installing ncmpcpp. It's a great command-line client for MPD. You could try opening this and seeing if the issue is caused when you have an Iris client connected/disconnected.

I'll give it a go.
